pkg/xds/bootstrap/generator.go

```diff
@@ -101,6 +106,22 @@ func (b *bootstrapGenerator) generateFor(proxyId core_xds.ProxyId, dataplane *co
 	return b.configForParameters(params)
 }
 
+func (b *bootstrapGenerator) verifyAdminPort(adminPort uint32, dataplane *core_mesh.DataplaneResource) error {
+	for _, inbound := range dataplane.Spec.Networking.Inbound {
+		if adminPort == inbound.Port {
+			return errors.Errorf("Resource precondition failed: Port %d requested as both admin and inbound port.", adminPort)
+		}
+	}
+
+	for _, outbound := range dataplane.Spec.Networking.Outbound {
+		if adminPort == outbound.Port {
+			return errors.Errorf("Resource precondition failed: Port %d requested as both admin and outbound port.", adminPort)
+		}
```

Comment:
same for outbound. It defaults to `127.0.0.1` but can have other address

Reply:
I see, will fix.